crates/shim/README.md

```diff
@@ -66,26 +66,28 @@ impl shim::Task for Service {
 }
 
 fn main() {
-    shim::run::<Service>("io.containerd.empty.v1")
+    shim::run::<Service>("io.containerd.empty.v2")
```

Comment:
`io.containerd.${shim-name}.vX` is ok. Both v1 and v2 are valid version of shim.
I think we should not confuse user here.

Reply:
I remember that there is a rule to map a shim name(`io.containerd.empty.v1`) to its binary(`containerd-shim-empty-v2`), the steps to run an empty that use the name with `containerd-shim-empty-v2`, so I think here should change to the same v2.

Reply:
Seems the containerd-shim crate targets shim-v2 only?

Reply:
The version is just version of shim implementation, not for the shim-v2 proto. containerd doesn't map v1 to v2 so that the `io.containerd.empty.v1` will be `containerd-shim-empty-v1`. The related code is [here](https://github.com/containerd/containerd/blob/7fa44fc98f4bde30fa154c361e11609f742aaccb/runtime/v2/shim/util.go#L92)